k8s/demo/common/wa/wa-proto-frontend.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: wa-proto-frontend
+  namespace: wa
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.nodejs: glob:prod-*
+spec:
+  releaseName: wa-proto-frontend
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/wa-proto-frontend
+  values:
+    nodejs:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      ingressClass: traefik-no-proxy
+      image: hmctspublic.azurecr.io/wa/wa-proto-frontend:latest
+      ingressHost: wa-prot-frontend.demo.platform.hmcts.net
```

Comment:
Looks like a typo here `wa-prot-frontend`

Reply:
Well, spot! It's fixed now. Cheers!